docs/_documentations/remote-configuring.md

```diff
@@ -0,0 +1,151 @@
+---
+layout: docs
+title: Configuring Codewind remote
+description: Configuring your environment to use Codewind remote
+keywords: users, projects, Kubernetes, LDAP, user management, access management, login, deployment, pod, security, securing Cloud connection
+duration: 5 minutes
+permalink: remoteconfiguring
+type: document
+parent: installoncloud
+order: 1
+---
+
+# Configuring Codewind remote
+
+To use Codewind remote, configure your system by following these steps. 
+
+## Prerequisites
+- Install your preferred IDE on your local machine. For more information about installing Eclipse, see [Getting started with Codewind for Eclipse](mdteclipsegettingstarted.html), or for more information about installing VS Code, see [Getting started with Codewind for VS Code](mdt-vsc-getting-started.html).
+- Ensure you have access to the Codewind CLI `cwctl`. To access the Codewind CLI... (**where???**)
+
+## Procedure
+
+To securely configure Codewind remote there are two options, configuring Kubernetes for Docker Desktop, or configuring a Cloud deployment that does not have an Ingress NGINX controller, for example, OpenShift. 
+
+### Configuring Kubernetes for Docker Desktop
+
+1. Validate you have an ingress controller installed on your cluster by typing the following command: [Mark to supply] 
+
+   If you are developing on Windows [get map local address from Mark]
+
+   If you do not have an ingress controller, install one using the following commands:
+   - `kubectl apply -f https://raw.githubusercontent.com/kubernetes/ingress-nginx/master/deploy/static/mandatory.yaml`
+   - `kubectl apply -f https://raw.githubusercontent.com/kubernetes/ingress-nginx/master/deploy/static/provider/cloud-generic.yaml`
+
+2. Create the remote deployment by entering the following `cwctl` command: 
+   `cwctl install remote \`
+     `--namespace {your_namespace}  \`
+     `--kadminuser admin \`
+     `--kadminpass <keycloakPassword>  \`
+     `--krealm codewind \`
+     `--kclient codewind  \`
+     `--kdevuser developer \`
+     `--kdevpass <userPassword>`
+
+### Configuring Cloud deployments that do not have an Ingress NGINX controller
+
+1. If you are deploying to a Cloud that does not have an Ingress NGINX controller, for example, OpenShift, you must provide the ingress value in the `cwctl install remote \` command during installation. Derive the `ingress` value from the cluster URL. [Needs more from Mark]
+2. Create the remote deployment by issuing the following example `cwctl` command: 
+
+  `cwctl install remote \`
+     `--namespace {your_namespace}  \`
+     `--kadminuser admin \`
```

Comment:
Explain whether the values to the "kadmin" and "kdev" arguments can be exactly what is written here, or depends on what the user has setup in his environment?

Reply:
Clarified command parameters